src/main/cpp/t07_quad2.cpp

```diff
@@ -21,5 +21,23 @@
 using namespace std;
 
 int t07_quad2() {
-
-};
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'd' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4311533)